### PR TITLE
Create Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by taldr27**
In this milestone I:
- Installed linters
- Created 3 different files `person.rb`, `student.rb` and `teacher.rb`
- Added a class in each of the files.
- Added a README file

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
```
Comment (cynthiainga) [inline student.rb]: - [x] You should add `age`, `name`, and `parent_permission` as parameters in `initialize then pass them in `super` 🙃 . Please kindly fix that 😉

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(10, name, parent_permission: true)
```
Comment (cynthiainga) [inline teacher.rb]: - [x] You should add `age`, `name`, and `parent_permission` as parameters in `initialize then pass them in `super` 🙃 . Please kindly fix that 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @taldr27, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 2 above)
Comment (taldr27) [reply teacher.rb]: Okay, thanks @cynthiainga for your review. I'll add those parameters to the method.

**Review by cynthiainga — APPROVED** [review]
Hi @taldr27, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️
- [ ] Correct Gitflow ✔️
- [ ]  Descriptive PR ✔️
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**